Sitka-Sound-Science-Center/sitka-landslide
# Added terraform and cipublish

Based on Dockerfile and other fixes from Klaas PR #39
    
It works to create everything needed for the site to be built
every 20 minutes (and it is).

We don't have any secrets for tfvars so we don't have one right
now.

There are some rough edges:
 - sitka/sitkaproduction are hardcoded in places
 - Three networking IDs were manually looked up #Manually
 - Just made one big site.tf terraform
 - The domain cert/autovalidation works but has to use us-east-1? (AWS bad
at making regions?)

Also drop github pages CI.

Resolves: #40


## Testing Instructions

 * Make some changes to the Copy and try the cipublish script
 * Check for any mistakes/cleanups in terraform
